Rakefile

```diff
@@ -21,6 +21,7 @@ namespace "test" do
   task xctool: %w[test:xctool:ios test:xctool:osx]
   namespace :xctool do
     desc "Run unit tests for all iOS targets using xctool"
+    run "echo Using xctool v`xctool -v`..."
```

Comment:
This'll be printed out every time the Rakefile is used, since it's not in a `do ... end` closure.

This could either be copy-pasted into the `test:xctool:ios` and `test:xctool:osx` tasks, or you could put this into its own task, like `test:xctool:version`, and have the `test:xctool:ios` and `test:xctool:osx` tasks dependent upon that task.